### ISSUE TITLE
Add support to Visual Studio 2022

### ISSUE THREAD

**Issue by BlinD-HuNTeR** [ReMinoer/SubSolution#1]
Visual Studio 2022 is now a 64-bit application, and will not install any extensions packaged for previous versions, even with open-ended versioning.

According to [Microsoft's documentation](https://docs.microsoft.com/en-us/visualstudio/extensibility/migration/update-visual-studio-extension), it's now required to specify 2 installation targets like this:
```
<Installation>
   <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0,17.0)">
      <ProductArchitecture>x86</ProductArchitecture>
   </InstallationTarget>
   <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[17.0,)">
      <ProductArchitecture>amd64</ProductArchitecture>
   </InstallationTarget>
</Installation>
```
Also I have tested this change by manually replacing the vsixmanifest with 7-zip and was able to install the extension. But there are still some erros in the execution (I see the log messages of solution building, but no dialog appears). Maybe I can debug these errors later, when I get some time.

Btw, this project is really awesome! I'm surprised there are no forks or any other open issues. This thing really deserves more attention.

**Comment by ReMinoer** [comment]
Hi @BlinD-HuNTeR :)

Thank you for your interest and your feedback. I took a bit of a pause on SubSolution but I will come back on it soon to release the extension on the Visual Studio Marketplace.

About the Visual Studio 2022 compatibility, it seems there are more changes to do to build it for 64 bits (see [Extensions WITH running code](https://docs.microsoft.com/en-us/visualstudio/extensibility/migration/update-visual-studio-extension?view=vs-2022#extensions-with-running-code)). I finally installed VS 2022 and I will try to make some tests soon.
